index.js

```diff
@@ -216,7 +216,8 @@ export const Rels = {
 		quiz: 'https://quizzes.api.brightspace.com/rels/quiz',
 		section: 'https://quizzes.api.brightspace.com/rels/quiz-section',
 		studyRecommendations: 'https://quizzes.api.brightspace.com/rels/studyRecommendations',
-		timing: 'https://quizzes.api.brightspace.com/rels/timing'
+		timing: 'https://quizzes.api.brightspace.com/rels/timing',
+		viewSubmissionResults: 'https://quizzes.api.brightspace.com/rels/view-quiz-submission-results'
```

Comment:
submission is an interesting language choice here, is there a reason we are using that instead of say viewAttemptResults.

I also don't think we need to use the word `quiz` in-between view and submission, we are already in the Quiz subdomain.

Reply:
I named it that way to be consistent with the other consts, which generally seem to match the end of the url route, e.g. `questionPool` for `https://quizzes.api.brightspace.com/rels/quiz-question-pool` etc. 🤔

Reply:
After offline discussion, we decided to name the rel const to `viewQuizSubmissionResults` for consistency